lib/plugins/splitIOFeatureManagerPlugin.ts

```diff
@@ -113,6 +113,7 @@ function plugin(fastify: FastifyInstance, opts: SplitIOOptions, done: () => void
     .init()
     .then(() => done())
     .catch(() => {
+      /* istanbul ignore next */
```

Comment:
🟥 I strongly believe it would be better to lower the coverage threshold than it would be to make the test coverage results lie.

Reply:
It's ok for us to conclude that some things aren't testable (or so difficult to test as to be not worth it), but our test coverage results should always be honest about that

Reply:
fixed